envoy/Makefile

```diff
@@ -110,20 +133,22 @@ $(ENVOY_API_SOURCES): $(ENVOY_API_PROTO_PATH)
 $(FILTER_TARGETS): $(FILTER_SOURCES)
 	$(PROTOC) --proto_path=$(ENVOY_API_PROTO_PATH) $(PROTO_DEPS) --go_out=plugins=grpc:$(GO_OUT) $(FILTER_SOURCES)
 
-install: force-root
+install:
+	make force-root
 	$(INSTALL) -m 0755 -d $(DESTDIR)$(BINDIR)
 	$(INSTALL) -m 0755 -T $(ENVOY_BIN) $(DESTDIR)$(BINDIR)/cilium-envoy
 # Strip only non-debug builds
 ifeq "$(findstring -dbg,$(realpath bazel-bin))" ""
 	$(STRIP) $(DESTDIR)$(BINDIR)/cilium-envoy
 endif
 
-bazel-archive: force-non-root tests clean-bins
+bazel-archive: tests clean-bins
```

Comment:
These targets are not used right now, I think, but still it does not make sense to build bazel caches as root.

Reply:
This one depended on `tests`, which ran the `force-non-root` target, so that's why I removed it here.